src/server/twitterSearch.js

```diff
@@ -0,0 +1,20 @@
+var Twit = require('twit')
+
+// These keys should be hidden in a private config file or environment variables
+// For simplicity of this assignment, they will be visible here
+var T = new Twit({
+  consumer_key:         'YiSLB0kOlsTd21UGYT32YOUgg',
+  consumer_secret:      '78b5VrGzkcIkpmftLdlFwirraelPRq2t5bFlgEcMkfaQqQh1Mb',

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Identifier 'consumer_secret' is not in camel case.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2161373217/issues/source?bid=3148782&fileBranchId=3159920#l7)